### PR TITLE
minSdkVersion="9"

### PR DESCRIPTION
**Pull request by JakeWharton**

### CODE REVIEW

**Comment by JakeWharton** [comment]
Lint... https://github.com/ReactiveX/RxAndroid/pull/206

File: build.gradle [hunk 1]
```diff
@@ -8,9 +8,8 @@ buildscript {
 }
 
 ext {
-    minSdkVersion = 15
+    minSdkVersion = 9
     compileSdkVersion = 22
-    targetSdkVersion = compileSdkVersion
     buildToolsVersion = '22.0.1'
     sourceCompatibility = JavaVersion.VERSION_1_7
 }
```
Comment (JakeWharton) [inline build.gradle]: Things that matter for the library are in this file.

File: sample-app/build.gradle [hunk 2]
```diff
@@ -5,8 +5,8 @@ android {
     buildToolsVersion rootProject.ext.buildToolsVersion
 
     defaultConfig {
-        minSdkVersion rootProject.ext.minSdkVersion
-        targetSdkVersion rootProject.ext.targetSdkVersion
+        minSdkVersion 15
+        targetSdkVersion 22
```
Comment (JakeWharton) [inline sample-app/build.gradle]: The sample doesn't really matter :anchor:

**Comment by dlew** [comment]
:+1:

**Pull request merged**